### ISSUE TITLE
Contact page is out of date

### ISSUE THREAD

**Issue by mauzey1** [PCMDI/cmor3_documentation#112]
The current [contact page](https://cmor.llnl.gov/mydoc_contacts/) for the CMOR documentation contains outdated info.  The last link under "CMIP6 table issues" is broken, and the mailing list info is for the defunct listserv email.  I will remove the broken link, and replace mailing list email with pcmdi-cmip@llnl.gov.

@durack1 @taylor13  Do you have other suggestsions?

![Screenshot 2022-08-18 at 16-00-35 Contact us CMOR 3](https://user-images.githubusercontent.com/42009431/185509692-e697e061-d193-40e4-96e8-730c025e466a.png)

**Comment by durack1** [comment]
@mauzey1 thanks for catching this. I am unfamiliar if our new Office365 group lists allow self-subscription, if yes, it would be great to replace the `subscribe cmor` advice, if not, I suggest removing this and replacing it with an email to the standard cmor@llnl.gov - the current mailing list.

It may also be useful to add the page https://wcrp-cmip.github.io/WGCM_Infrastructure_Panel//cmor_and_mip_tables.html which lists many of the `*mip*` tables that we know about, and their github entry point.

@matthew-mizielinski is there any other resources that we can point folks to?

**Comment by matthew-mizielinski** [comment]
> @matthew-mizielinski is there any other resources that we can point folks to?

I don't think so. I think @piotr-florek-mohc  and I need to put together a few notes on how to include the forecast_time coordinate.

**Comment by durack1** [comment]
@mauzey1 may as well catch this too - cmor@llnl.gov is the current mailing list address, which is still listserv and still working

**Comment by mauzey1** [comment]
@durack1 Are the instructions to get on the mailing list the same for cmor@llnl.gov as it was for the original listserv?

**Comment by durack1** [comment]
@mauzey1 to close out this issue, I suggest we remove the whole "CMIP6 table issues" section, and update the email address to cmor@llnl.gov (the lists. is redundant).

Also "CMOR3 documentation~s~ issues" (lose the s trailing documentation).

As far as I know the listserv info is correct - this hasn't changed for decades..

Then this issue can be closed
